pfcpiface/bess.go

```diff
@@ -20,8 +20,8 @@ import (
 )
 
 const (
-	// DefaultBurstSize for cbs, pbs required for dpdk  metering.
-	DefaultBurstSize = 2048
+	// DefaultBurstSize for cbs, pbs required for dpdk metering. 32 MTUs.
+	DefaultBurstSize = 32 * 1514
```

Comment:
This value now has the lowest priority: matching QCI > fallback QCI > `DefaultBurstSize`.
To me, increasing it to 50K still makes sense as it is used in the slice meters, which don't see flows. Thus the full batch hits the same meter.